### PR TITLE
BaSP-MR-74: Create logic route of activity

### PR DESCRIPTION
**Pull request by AleMac17**
- Split the resource in two screens (Form and Table)
- When I press the Add/Edit action button in the table it must redirect to the Form (Use React-Router hooks)
- When I finished to add or edit a Super Admin it must redirect me back to the table (Use React-Router hooks)

### CODE REVIEW

File: src/Components/Activities/Form/index.jsx [hunk 1]
```diff
@@ -1,68 +1,128 @@
...
+    }
+  }, []);
 
   const onChangeInput = (e) => {
     setActivity({
-      ...activities,
+      ...activity,
       [e.target.name]: e.target.value
     });
+    console.log('hola');
```
Comment (ghost) [inline src/Components/Activities/Form/index.jsx]: delete this console.log

**Review by ghost — APPROVED** [review]

**Review by Gonzarey96 — APPROVED** [review]
The code looks clean and easy to understand, congratulations!

**Comment by lorenzomauro59** [comment]
looks cleannn, I'ts approved!

**Review by lorenzomauro59 — APPROVED** [review]
looks cleaaann, It's approved!

**Comment by Federico-Peralta** [comment]
[[Activity] - Move the logic of the form to the new route](https://trello.com/c/7VJ9UNJP/74-activity-move-the-logic-of-the-form-to-the-new-route)

**Comment by nazarenapantanetti** [comment]
Tested and approved!

**Comment by nazarenapantanetti** [comment]
Tested and approved!

**Pull request merged**